docs/circuit_cutting/tutorials/gate_cutting_to_reduce_swap_overhead.ipynb

```diff
@@ -0,0 +1,301 @@
+      "text/plain": [
+       "<Figure size 1634.66x294.311 with 1 Axes>"
+      ]
+     },
+     "execution_count": 6,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "qpd_circuit = decompose_gates(circuit, [8, 20, 32])\n",
```

Comment:
It might be more instructive to programmatically generate these indices than to pull them out of thin air.

```suggestion
    "cut_indices = [\n",
    "    i for i, instruction in enumerate(circuit.data)\n",
    "    if {circuit.find_bit(q)[0] for q in instruction.qubits} == {0, 3}\n",
    "]\n",
    "qpd_circuit = decompose_gates(circuit, cut_indices)\n",
```